neurons/miners/blacklist.py

```diff
@@ -135,4 +136,6 @@ def base_blacklist(self, synapse: bt.Synapse) -> typing.Tuple[bool, str]:
         return True, f"Blacklisted hotkey: {hotkey}"
     if hotkey not in self.miner_config.whitelisted_hotkeys and self.config.mode == 'prod':
         return True, f"Not Whitelisted hotkey: {hotkey}"
+    if len(self.miner_config.inmemory_hotkeys) >= 1 and hotkey not in self.miner_config.inmemory_hotkeys and random.random() < self.miner_config.vali_penalty_rate:
```

Comment:
Now please add a condition flag, so we can remotely enable / disable this feature.
+ I need logs who would be punished even if feature is disabled